<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAbout</class>
 <widget class="QWidget" name="DialogAbout">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>关于</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url();
background-color: rgb(85, 170, 255);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QWidget" name="centralWgt" native="true">
     <widget class="QWidget" name="layoutWidgetDetails">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>786</width>
        <height>404</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="horizontalSpacing">
        <number>26</number>
       </property>
       <item row="3" column="1">
        <widget class="QLabel" name="label_11">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>金桥简黑体</family>
           <pointsize>20</pointsize>
           <weight>3</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgba(0, 0, 0, 255);
font: 25 20pt &quot;金桥简黑体&quot;;
background-color: rgba(255, 255, 127, 160);
border-radius: 10px; </string>
         </property>
         <property name="text">
          <string>010-62529086-8001/8006</string>
         </property>
         <property name="indent">
          <number>17</number>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>浪漫雅圆</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(0, 0, 0);
font: bold 20pt &quot;浪漫雅圆&quot;;
background-color: rgba(255, 255, 127, 160);
border-radius: 10px; 
</string>
         </property>
         <property name="text">
          <string>联系电话：</string>
         </property>
         <property name="margin">
          <number>14</number>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_SN">


         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>金桥简黑体</family>
           <pointsize>20</pointsize>
           <weight>3</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgba(0, 0, 0, 255);
font: 25 20pt &quot;金桥简黑体&quot;;
background-color: rgba(255, 255, 127, 160);
border-radius: 10px; </string>
         </property>
         <property name="text">
          <string>00:00:00:00</string>
         </property>
         <property name="indent">
          <number>17</number>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_7">
         <property name="font">
          <font>
           <family>浪漫雅圆</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(0, 0, 0);
font: bold 20pt &quot;浪漫雅圆&quot;;
background-color: rgba(255, 255, 127, 160);
border-radius: 10px; 
</string>
         </property>
         <property name="text">
          <string>版权所有：</string>
         </property>
         <property name="margin">
          <number>14</number>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_8">
         <property name="font">
          <font>
           <family>浪漫雅圆</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(0, 0, 0);
font: bold 20pt &quot;浪漫雅圆&quot;;
background-color: rgba(255, 255, 127, 160);
border-radius: 10px; 
</string>
         </property>
         <property name="text">
          <string>版  本 号：</string>
         </property>
         <property name="margin">
          <number>14</number>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label_14">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>金桥简黑体</family>
           <pointsize>20</pointsize>
           <weight>3</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgba(0, 0, 0, 255);
font: 25 20pt &quot;金桥简黑体&quot;;
background-color: rgba(255, 255, 127, 160);
border-radius: 10px; </string>
         </property>
         <property name="text">
          <string>北京中科虹霸科技有限公司</string>
         </property>
         <property name="indent">
          <number>17</number>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_13">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>金桥简黑体</family>
           <pointsize>20</pointsize>
           <weight>3</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgba(0, 0, 0, 255);
font: 25 20pt &quot;金桥简黑体&quot;;
background-color: rgba(255, 255, 127, 160);
border-radius: 10px; </string>
         </property>
         <property name="text">
          <string>business@irisking.com</string>
         </property>
         <property name="indent">
          <number>17</number>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_12">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>金桥简黑体</family>
           <pointsize>20</pointsize>
           <weight>3</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgba(0, 0, 0, 255);
font: 25 20pt &quot;金桥简黑体&quot;;
background-color: rgba(255, 255, 127, 160);
border-radius: 10px; </string>
         </property>
         <property name="text">
          <string>北京市海淀区中关村东路95号自动化大厦12层</string>
         </property>
         <property name="indent">
          <number>17</number>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="labVerValue">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>金桥简黑体</family>
           <pointsize>20</pointsize>
           <weight>3</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgba(0, 0, 0, 255);
font: 25 20pt &quot;金桥简黑体&quot;;
background-color: rgba(255, 255, 127, 160);
border-radius: 10px; </string>
         </property>
         <property name="text">
          <string>v1.0.0</string>
         </property>
         <property name="indent">
          <number>17</number>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_4">
         <property name="font">
          <font>
           <family>浪漫雅圆</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(0, 0, 0);
font: bold 20pt &quot;浪漫雅圆&quot;;
background-color: rgba(255, 255, 127, 160);
border-radius: 10px; 
</string>
         </property>
         <property name="text">
          <string>公司地址：</string>
         </property>
         <property name="margin">
          <number>14</number>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>浪漫雅圆</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(0, 0, 0);
font: bold 20pt &quot;浪漫雅圆&quot;;
background-color: rgba(255, 255, 127, 160);
border-radius: 10px; 
</string>
         </property>
         <property name="text">
          <string>序 列 号：</string>
         </property>
         <property name="margin">
          <number>14</number>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_6">
         <property name="font">
          <font>
           <family>浪漫雅圆</family>
           <pointsize>20</pointsize>
           <weight>75</weight>
           <italic>false</italic>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(0, 0, 0);
font: bold 20pt &quot;浪漫雅圆&quot;;
background-color: rgba(255, 255, 127, 160);
border-radius: 10px; 
</string>
         </property>
         <property name="text">
          <string>邮      箱：</string>
         </property>
         <property name="margin">
          <number>14</number>
         </property>
         <property name="indent">
          <number>10</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>285</x>
        <y>76</y>
        <width>621</width>
        <height>49</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>浪漫雅圆</family>
        <pointsize>36</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);
font: 36pt &quot;浪漫雅圆&quot;;</string>
      </property>
      <property name="text">
       <string>北京中科虹霸科技有限公司</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>140</y>
        <width>1024</width>
        <height>5</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: qlineargradient(spread:reflect, x1:0.534444, y1:0, x2:0, y2:0, stop:0 rgba(0, 0, 255, 255), stop:1 rgba(255, 255, 0, 255));
border-radius:10pt</string>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>288</x>
        <y>79</y>
        <width>3</width>
        <height>40</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>63</y>
        <width>151</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="ikembx00.qrc">:/image/iriskingLog.ico</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidgetTop">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>11</y>
        <width>1001</width>
        <height>52</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" columnstretch="1,4" rowminimumheight="0" columnminimumwidth="200,0">
       <item row="0" column="0">
        <widget class="QPushButton" name="toolBtnGoBack">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <pointsize>20</pointsize>
          </font>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">color: white; 
background-image: url(); 
background-color: rgb(32, 90, 167); 
border: 2px groove gray; 
border-radius: 10px; 
padding: 2px 2px; </string>
         </property>
         <property name="text">
          <string>返回</string>
         </property>
         <property name="icon">
          <iconset resource="ikembx00.qrc">
           <normaloff>:/image/jtbq_022.ico</normaloff>:/image/jtbq_022.ico</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>40</width>
           <height>40</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <spacer name="horizontalSpacer">
         <property name="font">
          <font>
           <pointsize>21</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:rgb(0, 0, 0)</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>448</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>199</x>
        <y>563</y>
        <width>651</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>文泉驿等宽微米黑</family>
        <pointsize>12</pointsize>
        <weight>50</weight>
        <bold>false</bold>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>声明：本系统所引用图片部分来源于网站，作者不详！</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>toolBtnGoBack</tabstop>
 </tabstops>
 <resources>
  <include location="ikembx00.qrc"/>
 </resources>
 <connections/>
</ui>
